mapie/quantile_regression.py

```diff
@@ -656,9 +656,9 @@ def predict(
         n = self.n_calib_samples
         q = (1 - (alpha)) * (1 + (1 / n))
 
-        y_preds = np.full(
+        y_preds = np.empty(
```

Comment:
Not sure why we are making this change? Why do we want to set the values to 0 as oppose to `np.nan`?